.travis.yml

```diff
@@ -1,22 +1,40 @@
+language: ruby
+rvm:
+  - 2.5.1
+cache:
+  yarn: true
+  directories:
+    - '.eslintcache'
+    - 'node_modules'
+
+install:
+  - bundle install
+  - nvm install node
+  - node -v
```

Comment:
We need show the version?

Reply:
I used the `webpacker` repo as reference for this file. I'm not sure why they do this. :)

Reply:
:smile_cat: